utils.js

```diff
@@ -43,11 +44,28 @@ class Cache extends Map {
   }
 }
 
-/** @type {import('.').Cache} */
-const cache = new Cache();
+const IDL_TYPES = new Set([
+  "_IDL_",
+  "attribute",
+  "dict-member",
+  "dictionary",
+  "enum-value",
+  "enum",
+  "exception",
+  "interface",
+  "method",
+  "typedef",
+]);
+
+const CONCEPT_TYPES = new Set(["_CONCEPT_", "dfn", "element", "event"]);
+
+const SUPPORTED_TYPES = new Set([...IDL_TYPES, ...CONCEPT_TYPES]);
 
 module.exports = {
```

Comment:
Should we use Node12 on the server ;)

Reply:
11 would work too. 12 won't show experiment warnings of fs.promises

Reply:
Alright, will update tomorrow.